flask/app/routes.py

```diff
@@ -162,3 +163,42 @@ def change_password():
     except:
         db.session.rollback()
         return 'Server error', 500
+
+
+@app.route('/api/participants/<participant_id>', methods = ['PATCH'])
+# @login_required # check document for how to test endpoints with login signing? 
+def update_participants(participant_id):
+    patch_json = request.get_json()
+
+    # convert to dictionary
+    patch_d = json.loads(patch_json)
+
+    # get the PK
+    id = patch_d['participant_id']
```

Comment:
Look in the admin routes for an example. Python dictionaries raise a KeyError if it's not found so you'll want to handle this case too.